recipes/rapidjson/config.yml

```
@@ -1,3 +1,5 @@
 versions:
   "1.1.0":
     folder: "1.1.x"
+  "20200410":
```

```suggestion
  "1.1.0-next.0":
```

Adding a `-identifier` states in semver that it is a pre-release
`next` because it should communicate that it is the _**next**_ version after _**1.1.0**_
Adding a _**dot-separated pre-release identifier**_ starting counting at `0` for the case that we need newer _**1.1.0-next**_ versions based on different project revisions for the case that the project still won't make a new release

___________

```
<valid semver> ::= <version core> "-" <pre-release>

<pre-release> ::= <dot-separated pre-release identifiers>

<dot-separated pre-release identifiers> ::= <pre-release identifier>
                                          | <pre-release identifier> "." <dot-separated pre-release identifiers>

<pre-release identifier> ::= <alphanumeric identifier>
                           | <numeric identifier>
```

https://semver.org